app/models/podcast_import.rb

```diff
@@ -197,13 +274,19 @@ def update_image(purpose, image_url)
       to_insert << series.images.build(upload: clean_string(image_url), purpose: purpose)
     end
 
-    story.images.destroy(to_destroy) if to_destroy.size > 0
```

Comment:
No longer need to destroy these?

Reply:
There is no story attribute here. I looked around and bisected,  Is it vestigial?

Reply:
Oh, interesting.  How did this ever work then?  I guess `to_destroy` has always been empty.